### DISCUSSION TITLE
Why is my favorite API gone is this version?

### DISCUSSION THREAD

**Discussion by registerrier** [registerrier/gammapy#5] [Q&A] [answered]
Dear gammapy devs, I have been using `gammapy.cool.shiny_function` of gammapy v0.7 for years, and I can't find it any longer since I installed the latest version. That's unfortunate, can you suggest a replacement?

**Comment by luca-giunti** [comment]
Same here!

**Reply by luca-giunti** [reply]
Ah no, that was supposed to be an answer, not a comment

**Comment by registerrier** [comment, marked as answer]
This feature is replaced by `GeneralClass.run()`. See the associated documentation.
